src/components/Forecast.tsx

```
@@ -13,6 +14,27 @@ interface ForecastProps {
 }
 
 export function Forecast({ forecast, activeTab, setActiveTab, isDark }: ForecastProps) {
+  // Extract hourly forecast items for animation
+  const hourlyItems = forecast.list.map(item => {
+    const temp = kelvinToCelsius(item.main.temp);
+    return {
+      ...item,
+      animatedTemp: useAnimatedValue(temp, 1000, 1)
```

ESLint

[error] React Hook "useAnimatedValue" cannot be called inside a callback. React Hooks must be called in a React function component or a custom React Hook function.

(react-hooks/rules-of-hooks)